src/StatusQ/Controls/StatusSwitch.qml

```diff
@@ -0,0 +1,66 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.14
+import QtGraphicalEffects 1.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Switch {
+    id: statusSwitch
+
+    indicator: Rectangle {
+        id: oval
+        implicitWidth: 52
+        implicitHeight: 28
+        x: statusSwitch.leftPadding
+        y: parent.height / 2 - height / 2
+        radius: 14
+        color: statusSwitch.checked ? Theme.palette.primaryColor1
+                               : Theme.palette.directColor8
+
+
+        Rectangle {
+            id: circle
+            y: 4
+            width: 20
+            height: 20
+            radius: 10
+            color: Style.current.white
+            layer.enabled: true
+            layer.effect: DropShadow {
+                width: parent.width
+                height: parent.height
+                visible: true
+                verticalOffset: 1
+                fast: true
+                cached: true
+                color: "#22000000"
```

Comment:
As per design, this is 12% black. How about we introduce at `dropShadowColor: getColor('black', 0.12)` in `ThemePalette` ?

Reply:
yeah dont see it. Will fix.